storage.md

```diff
@@ -138,6 +142,8 @@ const uploadUrl = await storage("students").getUploadUrl(
 );
 ```
 
+If the files you're going to upload is expected to be bigger than 6MB, you should default to retrieve the upload URL, and then use the `PUT` method to upload file directly to Ampt Storage in an asynchronous way. 
```

Comment:
"If the file you're uploading is expected to be larger than 6MB, you should create an upload URL and use the `PUT` method to upload the file directly to Ampt Storage."